tests/lms/views/decorators/test_h_api.py

```diff
@@ -162,73 +145,23 @@ def test_it_does_nothing_if_the_feature_isnt_enabled(
         wrapped.assert_called_once_with(pyramid_request, mock.sentinel.jwt)
         assert returned == wrapped.return_value
 
-    def test_it_does_nothing_if_the_course_group_already_exists(
-        self, create_course_group, context, models, pyramid_request, wrapped, hapi_svc
-    ):
-        models.CourseGroup.get.return_value = mock.create_autospec(
-            CourseGroup, instance=True
-        )
-
-        returned = create_course_group(pyramid_request, mock.sentinel.jwt, context)
-
-        hapi_svc.post.assert_not_called()
-        assert not pyramid_request.db.add.called
-        wrapped.assert_called_once_with(pyramid_request, mock.sentinel.jwt)
-        assert returned == wrapped.return_value
-
-    def test_it_posts_to_the_group_create_api(
+    def test_it_calls_the_group_update_api(
```

Comment:
I wonder if changing this test name to explicitly mention `PATCH` would help with ambiguity?